include/Gaffer/ThreadState.h

```diff
@@ -116,10 +116,8 @@ class GAFFER_API ThreadState
 
 		static const ThreadState &current();
 
-		const Context *context() const
-		{
-			return m_context;
-		}
+		const Context *context() const { return m_context; }
+		const Process *process() const { return m_process; }
```

Comment:
Oh, we're allowed to collapse down the braces when it makes things clearer? I think it's completely appropriate here, just wondering now where I might be allowed to do it.

Reply:
We seem to have quite a lot of these already, including some of yours in ImagePlug :

`grep -r '{.*return.*}' include`

If I were to take a stab at retrofitting some criteria to them I guess it'd be "reduces vertical space without sacrificing clarity, and we can't be bothered to put it in a `.inl` file instead".